server/game/core/Player.js

```diff
@@ -674,69 +657,20 @@ class Player extends GameObject {
             adjuster.canAdjust(playingType, card, target, ignoreType, penaltyAspect)
         );
         var costIncreases = matchingAdjusters
-            .filter((a) => a.getAmount(card, this) < 0)
-            .reduce((cost, adjuster) => cost - adjuster.getAmount(card, this), 0);
+            .filter((adjuster) => adjuster.direction === CostAdjustDirection.Increase)
+            .reduce((cost, adjuster) => cost + adjuster.getAmount(card, this), 0);
         var costDecreases = matchingAdjusters
-            .filter((a) => a.getAmount(card, this) > 0)
+            .filter((adjuster) => adjuster.direction === CostAdjustDirection.Decrease)
             .reduce((cost, adjuster) => cost + adjuster.getAmount(card, this), 0);
 
         baseCost += costIncreases;
         var reducedCost = baseCost - costDecreases;
 
-        var costFloor = Math.min(baseCost, Math.max(...matchingAdjusters.map((a) => a.costFloor)));
+        // TODO: not 100% sure what the use case for this line is
```

Comment:
I'm guessing it's for effects that say "reduce cost to a minimum of X". None of those in SWU, though of course there is always a minimum of 0.